src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -19,6 +19,7 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
   enterVenue,
 }) => {
   const { title, authorName, categories } = posterVenue.poster ?? {};
+  const number = posterVenue.name.replace("poster", "");
```

Comment:
Probably want to `useMemo` this so we don't have to do the string replace on every render.

Reply:
@sparkletown/product @mike-lvov Would be good to get your thoughts/input here for the more general codebase as well.

If we don't want this (or a variation of it) in our main codebase, then we can make this an `env/ohbm` specific 'hack'.

Reply:
> @yarikoptic I'm curious, would you also be happy with showing the full `posterVenue.name`, without this `replace`?

if needed - I am ok with such compromise. Indeed since DB-based value might not be generic enough, albeit it is a "poster hall". Can't fathom how this would take any notable amount of time though in comparison to the rest

Reply:
> Can't fathom how this would take any notable amount of time though in comparison to the rest

@yarikoptic If you mean when rendering, it's less about that, and more that poster pages aren't guaranteed to have the word poster in their `name`/`id`/etc. So the assumption that we can do a `name.replace('poster', '')` to wipe it out is invalid/not generalisable.

Reply:
stripped away my stripping of the poster.  It still looks good and indeed more factual -- who knows, might be a "demo" since says "Search posters/demos". So now the diff is really tiny ;-)